src/app/api/emilyDeposit/route.ts

```diff
@@ -11,12 +12,17 @@ interface CreateDepositRequestBody {
 export async function POST(req: NextRequest) {
   try {
     const body: CreateDepositRequestBody = await req.json();
+    const transactionHex = await getTransactionHex(body.bitcoinTxid);
```

Comment:
If `getTransactionHex` fails, will this return a "nice" error to the api caller or just a generic 500?

Reply:
it should return the error from the response